e2e/commands_test.go

```diff
@@ -405,6 +407,15 @@ func initializeDockerAppEnvironment(t *testing.T, cmd *icmd.Cmd, tmpDir *fs.Dir,
 	icmd.RunCmd(*cmd).Assert(t, icmd.Success)
 }
 
+func assertAppLabels(t *testing.T, cmd *icmd.Cmd, appName, containerName string) {
+	cmd.Command = dockerCli.Command("inspect", fmt.Sprintf("%s_%s", appName, containerName))
+	checkContains(t, icmd.RunCmd(*cmd).Assert(t, icmd.Success).Combined(),
+		[]string{
+			fmt.Sprintf(`"%s": "%s"`, internal.LabelAppNamespace, appName),
+			fmt.Sprintf(`"%s": ".+"`, internal.LabelAppVersion),
```

Comment:
Note the expected regex does not include `internal.Version` as the version is changed between the builds and the coverage tests during CI which will cause it to fail